README.md

```diff
@@ -30,6 +30,40 @@ We use Jest with React Testing Library to write unit tests. For larger pieces of
 
 Before opening a pull request, you can run `npm run verify:local` to make sure your changes pass automated tests (Jest and Cypress) and linter (both JS and CSS linters). We also execute [husky](https://typicode.github.io/husky/) hooks with every commit to make sure the changes pass basic lint checks.
 
+## Testing with Playwright
+
+Copy the example env file and create a file named:.env For local development only the BASE_URL:https://stage.foo.redhat.com:1337 is required, which is already set in the example config.
+You also need to add the username and password.
```

Comment:
You also need to add the username and password of your Stage testing account to the `.env` file.